Cargo.toml

```diff
@@ -49,6 +49,7 @@ risc0-r0vm = { version = "0.19.0-alpha.1", default-features = false, path = "ris
 risc0-sys = { version = "0.19.0-alpha.1", default-features = false, path = "risc0/sys" }
 risc0-zkp = { version = "0.19.0-alpha.1", default-features = false, path = "risc0/zkp" }
 risc0-zkvm = { version = "0.19.0-alpha.1", default-features = false, path = "risc0/zkvm" }
+risc0-zkvm-methods = { version = "0.19.0-alpha.1", default-features = false, path = "risc0/zkvm/methods" }
```

Comment:
This breaks the release process, I use the `cargo release` tool and I have not yet found a way to publish methods crates.

Reply:
It breaks the release process to include this as a workspace level dependency? In it's Cargo.toml `release` is set to false, so it shouldn't try to be released if I understand correctly. Maybe you could clarify the issue?

Reply:
You can try it out with:
```
cargo release publish
```

This is safe to do because it will just do a dry run by default.

Reply:
So if I comment out the `package.metadata.release.release  = false` setting in the `Cargo.toml` for `risc0-zkvm-methods` and run `cargo release publish -p risc0-zkvm-methods` it fails with a build error, and I assume this is the error that prevents publishing methods packages.

When that line is left intact, `cargo release publish -p risc0-zkvm-methods` gives `error: no packages selected`. Additionally `cargo release publish` makes no mention of the methods package. So I _think_ the methods crate is being properly ignored, and this change will not change that?